ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -206,6 +206,10 @@ tests:
   steps:
     cluster_profile: aws
     workflow: origin-e2e-aws
+- as: e2e-aws-ovn
```

Comment:
I wish I'd caught this previous to merge.  We now run approximately 10 clusters on Origin.  That is not sustainable.